client/app/components/ApplicationArea/ApplicationLayout/VersionInfo.jsx

```diff
@@ -14,8 +14,8 @@ export default function VersionInfo() {
         <div className="m-t-10">
           {/* eslint-disable react/jsx-no-target-blank */}
           <Link href="https://version.redash.io/" className="update-available" target="_blank" rel="noopener">
-            Update Available
-            <i className="fa fa-external-link m-l-5" />
+            Update Available <i className="fa fa-external-link m-l-5" aria-hidden="true" />
+            <span className="sr-only">(opens in a new tab)</span>
```

Comment:
Is there an aria attribute (e.g. `aria-label`) that might be clearer?

Reply:
An `aria-label` would have to be added to the parent anchor, which kind of messes the meaning, because it would be read before the inner content. Also, it is not recommended that aria-labels be added to elements that have content which represent their meaning. The intent of the visually hidden span is to replace the meaning of the icon itself, which cannot be done with an aria-label, as the icon need to be hidden from the SR (otherwise there is a risk it'll be read as a random char). Take a look the [guide from Font Awesome](https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility).

Reply:
There is actually one further benefit from avoiding conveying meaning with attributes: it's way easier to i18n a span than an attribute (as of now it requires a hook).